apps/web/Dockerfile

```diff
@@ -1,5 +1,6 @@
 # start build stage
-FROM nikolaik/python-nodejs:python3.10-nodejs20-alpine as builder
+FROM node:20-alpine3.19 as builder
+RUN apk add g++ make py3-pip
```

Comment:
Did you test this without python?

I am unsure if we need python in this container

Reply:
I assumed that python was needed where basic images include python. Also Pawel said that python is used on the build and in some images the ARM architecture build hung without it. I will recheck and let you know 😊

Reply:
@AliaksandrRyzhou Lets double check this, I know its needed for inbound-mail for sure but I have not seen anything else in the project that needs it.

@LetItRock Are you aware of any build tools that leverage python for our project?

I want to make sure we are only doing what we need to cut build times down as much as possible, but @AliaksandrRyzhou if this takes more then 2 hours to debug lets create a ticket and look at it in the future so we can get this moved through.

Reply:
This I get when I try to build the web service without Python
![image](https://github.com/novuhq/novu/assets/141645418/24053579-645a-467b-bad9-1c7961a5e78c)